src/aviary/core.py

```diff
@@ -53,8 +53,6 @@
     "EvalAnswerMode",
     "Frame",
     "FunctionInfo",
-    "MalformedMessageError",
-    "Message",
```

Comment:
I'm a heavy user of `from aviary.core import everything, I, need` - I think it is very convenient. Could we keep these here? Something like:
```
from llmclient import Message

__all__ = [
     ...
    "Message",
    ...
]
```